Specifications/QIR/Data-Types.md

```diff
@@ -1,42 +1,11 @@
 ## Data Type Representation
 
 We define LLVM representations for a variety of classical and quantum data types.
+QIR does not expect the runtime to provide garbage collection. Instead, it specifies a set of runtime functions that can be used by the language specific compiler to implement a reference counting scheme if needed, if the source language requires automatic memory management. See the section of [reference and alias counting](#reference-and-alias-counting) for more detail.
```

Comment:
I might say "require" instead of "expect". I might also add a sentence saying that a runtime implementation that provides fully automated garbage collection is free to ignore (or strip out) the reference counting calls.

Reply:
Changed to require, and the sentence you suggested is already there further below in the section.